README.md

```diff
@@ -0,0 +1,91 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budgeti App
+
+> Budgeti is a budget tracking app that helps you monitor how your budget was allocated and spent. Budgeti is built with Ruby on Rails and styled using boostrap
+
+![mobile interface](https://user-images.githubusercontent.com/71140133/171690486-db241bbd-fe14-439a-85cd-116138865355.PNG)
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+- RSpec
+- Boostrap
+
+
+## Live Demo (if available)
```

Comment:
- [ ] [REQUIRED] Kindly ensure this project is deployed and accessible online using Heroku, Netlify, or GitHub Pages so that the project requirements can be met. Remember to add the deployment link to the README.